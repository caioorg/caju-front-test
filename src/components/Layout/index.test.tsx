import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import { Layout } from "."

describe("[Component] - Layout", () => {
  it("should render the header and children", () => {
    const testContent = "Test Content"
    const { asFragment } = render(
      <Layout>
        <div>{testContent}</div>
      </Layout>
    )

    expect(screen.getByText("Caju Front Teste")).toBeInTheDocument()
    expect(screen.getByText(testContent)).toBeInTheDocument()
    expect(asFragment()).toMatchSnapshot()
  })
})
