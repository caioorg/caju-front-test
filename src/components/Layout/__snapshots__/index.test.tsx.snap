// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] - Layout should render the header and children 1`] = `
<DocumentFragment>
  <header
    class="sc-braxZu ctnmxR"
  >
    <h1>
      Caju Front Teste
    </h1>
  </header>
  <section
    class="sc-gJhJTp dXojQS"
  >
    <div>
      Test Content
    </div>
  </section>
</DocumentFragment>
`;
